bin/run_tests

```diff
@@ -187,18 +183,14 @@ test_poet_core(){
     run_docker_test ./consensus/poet/core/tests/test_consensus/unit_consensus.yaml -s poet
 }
 
-test_poet_families(){
-    run_docker_test ./consensus/poet/families/tests/test_validator_registry/tp_validator_registry.yaml \
-    -s validator
-}
-
 test_poet_simulator(){
     run_docker_test ./consensus/poet/simulator/tests/test_simulator/unit_simulator.yaml -s poet
 }
 
 test_python_sdk() {
     # transaction processors
     run_docker_test tp-config -s validator
+    run_docker_test tp-validator-registry -s validator
```

Comment:
There are a variety of criteria by which the tests can be grouped. My thought was that all the TP tests should be grouped together because they use the same test apparatus. It doesn't matter much, and we can scrap this PR if we decide that's not how it should be.